docs/implementations/Readme.md

```diff
@@ -1,3 +1,5 @@
 # Implementations
 
 [CCF](./ccf.md) - Cloud Carbon Footprint Impact Plugin Documentation
+
+[WattTime](./watttime.md) - WattTime Impact Plugin Documentation
```

Comment:
you can add all other implementation links here too

Reply:
I will add a separate PR with others.